scripts/validate-artifacts.ps1

```diff
@@ -47,13 +47,52 @@ if ($virtualMachine) {
     throw "Unable to find Virtual Machine in the task resource group. Please make sure that you created the Virtual Machine and try again."
 }
 
-if ($virtualMachine.location -ne "uksouth" ) { 
+if ($virtualMachine.location -eq "uksouth" ) { 
     Write-Output "`u{2705} Checked Virtual Machine location - OK."
 } else { 
     Write-Output `u{1F914}
-    throw "Virtual is not deployed to the UK West region. Please migrate VM to another region and try again."
+    throw "Virtual is not deployed to the UK South region. Please re-deploy VM to the UK South region and try again."
```

Comment:
The script checks if the Virtual Machine is deployed in the 'uksouth' region. Ensure that this region is correct for your deployment requirements.